app/assets/stylesheets/style.css.scss

```diff
@@ -1156,18 +1156,21 @@ div.center {
   margin-top: 8px;
 }
 
-#flash_alert.alert,
-#flash_error.error,
-#flash_notice {
-  color: #ce4844;
+#flash_error,
```

Comment:
I am wondering if it would be more idiomatic to change all these to classes instead of IDs, since IDs are meant to be only used once. What do you think?